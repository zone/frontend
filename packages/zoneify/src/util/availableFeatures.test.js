const availableFeatures = require('./availableFeatures')

describe('availableFeatures', () => {
  it('returns all available features', () => {
    expect(availableFeatures).toMatchSnapshot()
  })
})
