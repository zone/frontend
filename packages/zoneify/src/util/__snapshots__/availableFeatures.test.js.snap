// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`availableFeatures returns all available features 1`] = `
Array [
  Object {
    "id": "browserslist",
    "name": "Browserslist",
    "questions": undefined,
    "run": [Function],
  },
  Object {
    "id": "editorConfig",
    "name": "EditorConfig",
    "questions": undefined,
    "run": [Function],
  },
  Object {
    "id": "eslint",
    "name": "ESLint",
    "questions": Array [
      Object {
        "choices": Array [
          Object {
            "name": "None",
            "value": "zone",
          },
          Object {
            "name": "React",
            "value": "react",
          },
          Object {
            "name": "Vue",
            "value": "vue",
          },
        ],
        "message": "What type of JS framework are you using?",
        "name": "framework",
        "type": "list",
      },
    ],
    "run": [Function],
  },
  Object {
    "id": "stylelint",
    "name": "stylelint",
    "questions": undefined,
    "run": [Function],
  },
]
`;
